js/main.js

```diff
@@ -1492,13 +1417,18 @@ function getBattleElement(taskName) {
     return document.getElementById(task.baseData.progressBarId);
 }
 
-function getItemElement(itemName) {
-    const item = gameData.itemData[itemName];
-    if (item == null) {
-        console.log('Item not found in data: ' + itemName);
+function getItemElement(name) {
+    const pointOfInterest = pointsOfInterest[name];
+    if (pointOfInterest == null) {

```

Comment:
Elemente die nicht gefunden werden sind undefined, nicht null (danke JS)
und bitte === verwenden, sonst wandelt JS alle möglichen Typen in andere Typen um, was es sehr schwer macht, vorherzusagen, was da eigentlich passiert.

Reply:
Jo, aber schau dir mal den vorherigen Code an.

Reply:
puh, das war doch auch müll 😅